app/lib/schema-data.ts

```diff
@@ -52,19 +60,34 @@ export async function loadJson(filePath: string): Promise<Schema> {
   return body
 }
 
-async function loadSubSchema(schemaArray: ReferencedSchema[]) {
+async function loadContentFromGithub(path: string, ref?: string) {
+  const ghData = (
+    await octokit.repos.getContent({
+      ...repoData,
+      path: path.replaceAll('\\', '/'),
+      ref: ref ?? 'main',
+    })
+  ).data as GitContentDataResponse
+  if (!ghData.content) throw new Response(null, { status: 404 })
+
+  return JSON.parse(Buffer.from(ghData.content, 'base64').toString()) as Schema
```

Comment:
I'm surprised that it returns the content base64-encoded. Can you avoid the decoding step by setting the [media type](https://github.com/octokit/octokit.js#media-type-formats)?